### PR TITLE
242903 bpc checkout clean

### PR DESCRIPTION
**Pull request by GodsonLeigh**
clean copy of bps checkout page
content change
Layout to re-introduce 2024 to scheme years

### CODE REVIEW

File: app/routes/models/calculation-bps.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  data.push({
+    schemeYear: schemeYears[index],
+    text: toCurrencyString(overallResult[property]),
+    format: 'numeric',
+    classes: 'govuk-body govuk-!-font-weight-bold'
+  })
+  return data
+}
+
+function createBackLink () {
```
Comment (AbidemiAdio) [inline app/routes/models/calculation-bps.js]: This function is redundant, hence it can be removed.

File: app/routes/models/calculation-bps.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+const schemeYears = require('../../calculation/scheme-years')
+const toCurrencyString = require('../../utils/to-currency-string')
+
+function ViewModel (value, calculations) {
+  this.model = {
+    confirmation: createSummary(value),
+    paymentBand: createTableDefinition(calculations, { property: 'rate', text: '', caption: 'Progressive reductions by scheme year', formatType: 'percentage', showOverall: false }),
+    reduction: createTableDefinition(calculations, { property: 'reduction', text: 'Total progressive reduction:', caption: 'Progressive reductions applied to your claim by scheme year', formatType: 'currency', showOverall: true }),
+    paymentSummary: createPaymentSummary(calculations, { caption: 'Estimated BPS payments 2021 to 2023' }),
+    backLink: createBackLink()
```
Comment (AbidemiAdio) [inline app/routes/models/calculation-bps.js]: There is no reference to this property in the calling model, hence it is redundant and should be removed.

File: app/routes/models/calculation-bps.js (hunk 1 above)
Comment (GodsonLeigh) [reply app/routes/models/calculation-bps.js]: redundant function and reference removed

File: app/routes/models/calculation-bps.js (hunk 2 above)
Comment (GodsonLeigh) [reply app/routes/models/calculation-bps.js]: redundant reference removed

**Review by AbidemiAdio — APPROVED** [review]
QED

**Pull request merged**